### DISCUSSION TITLE
Download entire playlist

### DISCUSSION THREAD

**Discussion by nyhilium** [Tzahi12345/YoutubeDL-Material#1033] [Q&A]
Hi. I have installed yt-dl-material as a Docker container on Unraid. Fetching single videos works perfectly, but i struggle with downloading entire playlists.
URL: https://www.youtube.com/playlist?list=PLohYzz4btpaRkjnf0BttlN_9OlIqzFixn
Audio only

After some time the status changes from "Getting Info" to error with following output:
`Error while retrieving info on video with URL https://www.youtube.com/playlist?list=PLohYzz4btpaRkjnf0BttlN_9OlIqzFixn with the following message: Error: Command failed with exit code 1: node_modules/youtube-dl/bin/youtube-dl -i -o audio/%(title)s.%(ext)s --write-info-json --audio-quality 0 -x --audio-format mp3 --write-thumbnail --no-clean-info-json -j --dump-json https://www.youtube.com/playlist?list=PLohYzz4btpaRkjnf0BttlN_9OlIqzFixn

WARNING: [youtube:tab] YouTube said: INFO - 24 unavailable videos are hidden
ERROR: [youtube] 5INYvb_MYtQ: Video unavailable. This video is not available
ERROR: [youtube] ShNGwaN3Tb0: Video unavailable. This video is not available
ERROR: [youtube] taWnY1I6zH0: Video unavailable. This video is not available
ERROR: [youtube] r6ZiKSNeuCs: Video unavailable. This video is not available
ERROR: [youtube] m_GzZgohZCo: Video unavailable. This video is not available
ERROR: [youtube] F2aSC9T194g: Video unavailable. This video is not available
ERROR: [youtube] ILILcRn5bj8: Video unavailable. This video is not available
ERROR: [youtube] 7Db0Mt6EVt8: Video unavailable. This video is not available
ERROR: [youtube] RA_2XbPhgu4: Video unavailable. This video is not available
ERROR: [youtube] 4tP2h0hqLy4: Video unavailable. This video is not available
ERROR: [youtube] -jElqiTosBA: Video unavailable. This video is not available
ERROR: [youtube] fiGUtCHa7HE: Video unavailable. This video is not available
ERROR: [youtube] Lau9FhKJGjc: Video unavailable. This video is not available
ERROR: [youtube] zkFs91DBS1Q: Video unavailable. This video is not available
ERROR: [youtube] GlL-B4omvC8: Video unavailable. This video is not available
ERROR: [youtube] nhmTk4b0ae4: Video unavailable. This video is not available
ERROR: [youtube] dc1jzdnJJWs: Video unavailable. This video is not available
ERROR: [youtube] cZ9CwHsd8u4: Video unavailable. This video is not available
ERROR: [youtube] 3A6TDRblzns: Video unavailable. This video is not available
ERROR: [youtube] FLtgtjrWZuw: Video unavailable. This video is not available
ERROR: [youtube] WDCsz_Muec0: Video unavailable. This video is not available
ERROR: [youtube] WAtMZxYikYs: Video unavailable. This video is not available
ERROR: [youtube] 84mpw5uaAKw: Video unavailable. This video is not available
ERROR: [youtube] hz2aWM-1ZHc: Video unavailable. This video is not available`

I know that there are 24 videos not showing due to an error, but is there no possibility to skip those and download only the ones that are available?

**Comment by Tzahi12345** [comment]
This isn't supposed to happen but I've seen it occur and other people have faced it. It's not always consistent but I have a PR up with some bug fixes (#1032) so I'll see if I can get this one squared away too.
